saleor/graphql/api.py

```diff
@@ -50,6 +53,16 @@ class Query(graphene.ObjectType):
         Collection, query=graphene.String(
             description=DESCRIPTIONS['collection']),
         description='List of the shop\'s collections.')
+    menu = graphene.Field(
+        Menu, id=graphene.Argument(graphene.ID),
+        description='Lookup a menu by ID.')
+    menus = DjangoFilterConnectionField(
+        Menu, description="List of the shop\'s menus.")
+    menu_item = graphene.Field(
```

Comment:
I'm not sure if we need to access menu items directly, do we? I guess it would be enough and more common to fetch the menu object and access its items from there.

Reply:
As far as I remember we had a discussion about it and we stated that we may want to have this endpoint, so we can open `Edit menu` view and assign items created before to the menu - similar to how we currently manage `attributes` of `Product Types`.
Did our assumptions change?

Reply:
I guess they didn't. It's fine if there's a valid use-case.